packages/expo-dev-launcher/android/src/main/java/expo/modules/devlauncher/launcher/DevLauncherActivity.kt

```diff
@@ -102,7 +102,29 @@ class DevLauncherActivity : ReactActivity(), ReactInstanceManager.ReactInstanceE
   }
 
   private val isSimulator
```

Comment:
expo-dev-launcher depends on expo-dev-menu -- one of the people who works on these modules can probably say whether expo-dev-menu could expose a Java-only method that expo-dev-launcher could use.

Reply:
While I agree in theory, to keep the change minimal all I really did was just swap out one condition check for a slightly larger one. Your proposed refactoring goes a bit further and could potentially wait for the upcoming audit to keep this PR lean.